src/reanimated2/platform-specific/webAnimations.ts

```diff
@@ -0,0 +1,632 @@
+import {
+  AnimationData,
+  AnimationsData,
+  TransformProperties,
+  WebEasings,
+} from './webAnimationsData';
+
+export const WEB_ANIMATIONS_ID = 'webAnimationsStyle';
+
+export interface AnimationConfig {
+  animationName: string;
+  duration: number;
+  delay: number;
+  easing: string;
+}
+
+/**
+ *  Creates `HTMLStyleElement`, inserts it into DOM and then inserts CSS rules into the stylesheet.
+ *  If style element already exists, nothing happens.
+ */
+export function insertWebAnimations(): void {
+  if (document.getElementById(WEB_ANIMATIONS_ID) !== null) {
+    return;
+  }
+
+  const style = document.createElement('style');
+  style.id = WEB_ANIMATIONS_ID;
+
+  document.head.appendChild(style);
+
+  for (const animationName in Animations) {
```

Comment:
```suggestion
  for (const animationType in Animations) {
```

to make it more consistent